test/gpu/native/PREDIFF

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+awk '!/Unknown CUDA version/ {print};' $2 > $2.no_version_warn
```

Comment:
Nice. It's a bit of a hack but I don't have any suggestions for what the real solution is. I'm hoping it goes away if\when we upgrade to a new version of LLVM or CUDA.

I think we would want to find a proper fix before the next release (6 months from now) or whenever GPU support gets to a more mature point. Do we have an open issue for this? If not can you open one just so we don't forget about it?

Reply:
I think we'll have to live with this until clang starts supporting newer CUDA versions. If anything, that increases the importance of fixing this. We can suppress them directly, or we can have some more strict version checking at Chapel build time to error if the version mismatch is in the other direction (too old CUDA).

Anyways, I have attempted suppressing the warning in a short period of time but wasn't able to for some reason. I created https://github.com/Cray/chapel-private/issues/2492